### PR TITLE
Release hammer_cli and hammer_cli_foreman 3.0.0

### PR DESCRIPTION
**Pull request by ofedoren**
@ekohl

Supported Versions:

 * Nightly

### CODE REVIEW

File: packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec [hunk 1]
```diff
@@ -5,8 +5,6 @@
 %global gem_name hammer_cli
 
 %global release 1
-%global prereleasesource pre.develop
```
Comment (ekohl) [inline packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec:8]: You can comment this

File: packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec [hunk 2]
```diff
@@ -5,8 +5,6 @@
...
-%global prerelease %{?prereleasesource:.}%{?prereleasesource}
```
Comment (ekohl) [inline packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec:9]: Please keep this

**Review by ekohl — CHANGES_REQUESTED** [review]
Same comments for the other spec.

**Comment by ofedoren** [comment]
@ekohl, updated. Did I understand you right?..

**Review by ekohl — COMMENTED** [review]
Yes, you understood me right, but I think this is actually an edge case. For nightly packages we don't release the GA build to nightly. What I think is right is if we get to the point where we branch, then update `rpm/develop` to `3.1.0-develop` (including hammer) and then update to 3.0.0 GA in `rpm/3.0`. Does that make sense?

**Comment by upadhyeammit** [comment]
> Yes, you understood me right, but I think this is actually an edge case. For nightly packages we don't release the GA build to nightly. What I think is right is if we get to the point where we branch, then update `rpm/develop` to `3.1.0-develop` (including hammer) and then update to 3.0.0 GA in `rpm/3.0`. Does that make sense?

Agree, can't see better way to handle this.

**Comment by tbrisker** [comment]
IIUC we agreed to stick with nightly hammer builds in rpm/develop, closing. If I misunderstood feel free to reopen.

**Pull request closed without merging**